contracts/sol6/KyberNetwork.sol

```diff
@@ -1026,15 +1025,15 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         require(input.src != input.dest, "src = dest");
 
         if (input.src == ETH_TOKEN_ADDRESS) {
-            require(msg.value == input.srcAmount, "bad eth qty");
+            require(msg.value == input.srcAmount); // kyberProxy issues message here
```

Comment:
The new proxy does, but the current proxy doesn't issue revert message for this case

Reply:
Yes

Got it